src/core/core.controller.js

```diff
@@ -1085,12 +977,12 @@ helpers.extend(Chart.prototype, /** @lends Chart */ {
 			me._lastEvent = null;
 		} else {
 			me.active = me.getElementsAtEventForMode(e, hoverOptions.mode, hoverOptions);
-			me._lastEvent = e.type === 'click' ? null : e;
+			me._lastEvent = e.type === 'click' ? me._lastEvent : e;
 		}
 
 		// Invoke onHover hook
 		// Need to call with native event here to not break backwards compatibility
-		helpers.callback(options.onHover || options.hover.onHover, [e.native, me.active], me);
+		// helpers.callback(options.onHover || options.hover.onHover, [e.native, me.active], me);
```

Comment:
this should probably be uncommented or removed

Reply:
Restored